priv/templates/elixir/controller.ex

```diff
@@ -21,6 +21,11 @@ defmodule <%= module %>Controller do
   @pagination_distance 5
 
   def index(conn, params) do
+    params = Map.put_new(params, "sort_direction", "<%= sort_direction %>")
+    params = Map.put_new(params, "sort_field", "<%= sort_field %>")
```

Comment:
This would be a good place to have a pipeline.

```elixir
params =
  params
  |> Map.put_new("sort_direction", "<%= sort_direction %>")
  |> Map.put_new("sort_field", "<%= sort_field %>")
```